release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -178,17 +178,42 @@ Limit Calculator] to estimate cluster limits for your environment.
 [id="ocp-4-3-storage"]
 === Storage
 
-[id="ocp-4-3-iscsi"]
-==== iSCSI
+[id="ocp-4-3-ocs-4-2"]
+==== OpenShift Container Storage 4.2
 
-Persistent volumes and raw block volumes using iSCSI, previously in Technology
-Preview, are now fully supported in {product-title} 4.3.
+Deploy, manage, monitor, and migrate a Red Hat OpenShift Container Storage 4.2 cluster. See the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html/4.2_release_notes/[Red Hat OpenShift Container Storage 4.2 Release Notes].
+
+[id="ocp-4-3-iscsi-persistent-storage"]
+==== Persistent storage Using iSCSI
+
+Persistent volumes using iSCSI, previously in Technology Preview, is now fully supported in {product-title} {product-version}.
```

Comment:
You should hard code 4.3 here, since it is actually just 4.3 and would not make sense if this ever got carried forward to 4.4.

Reply:
Thanks, was wondering about that!